modules/kafka/src/main/scala/com.snowplowanalytics.snowplow.enrich.kafka/Source.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.snowplow.enrich.kafka
+
+import cats.effect.{ConcurrentEffect, ContextShift, Timer}
+import com.snowplowanalytics.snowplow.enrich.common.fs2.config.io.Input
+import fs2.kafka.{CommittableConsumerRecord, ConsumerSettings, KafkaConsumer}
+import fs2.Stream
+
+object Source {
+
+  def init[F[_]: ConcurrentEffect: ContextShift: Timer](
+    input: Input
+  ): Stream[F, CommittableConsumerRecord[F, String, Array[Byte]]] =
+    input match {
+      case k: Input.Kafka => kafka(k)
+      case i => Stream.raiseError[F](new IllegalArgumentException(s"Input $i is not Kafka"))
+    }
+
+  def kafka[F[_]: ConcurrentEffect: ContextShift: Timer](
+    input: Input.Kafka
+  ): Stream[F, CommittableConsumerRecord[F, String, Array[Byte]]] = {
+    val consumerSettings =
+      ConsumerSettings[F, String, Array[Byte]]
+        .withBootstrapServers(input.bootstrapServers)
+        .withEnableAutoCommit(false)
+        .withProperties(input.consumerConf)
```

Comment:
At first I wanted to suggest that :
```suggestion
        .withProperties(input.consumerConf)
        .withBlocker(blocker)
```
but the documentation says:
```
Because the underlying Java consumer is not thread-safe, the blocker must be single-threaded. If in doubt, leave this unset so that a default single-threaded blocker will be provided.
```
and we don't want to make our blocking thread pool single-threaded so I think that we should leave it as is.

Reply:
yeah, I also went through the same thinking and ended up leaving it as is